README.md

```diff
@@ -1,13 +1,22 @@
 # Meetups App
 
-## First run
-1. Create `.env` file in the root dir with fields from `.env.example`
-2. Optionally (to use maps) [get and set](https://yandex.ru/dev/jsapi-v2-1/doc/ru/#get-api-key) yandex maps api key or, even better, [google maps api key](https://developers.google.com/maps/documentation/javascript/get-api-key) in the `.env` file
-3. Install docker-compose
-4. `chmod +x entrypoint.sh && ./entrypoint.sh` (You can also use entrypoint-mac.sh if you are using mac, but be sure to install psycopg lib before that)
+## Basic setup
+1. Create `.env` file in the root dir with filled out fields from `.env.example`
+2. Install docker-compose for setting up local postgres db and application itself
+3. Run `docker-compose -f deployments/docker-compose.yml up -d`
+4. Go to `http://localhost:8000/swagger` and you should see the app running
 
-This will install postgres db in docker, install geo dependencies, create superuser admin:admin, run migrations and run server.
-5. After you have done this once you can just run `python manage.py runserver` next time
+## First run for local development (will install dependencies locally)
```

Comment:
My suggestion is to completely do away with local launching not in docker
And not even to describe such steps in order not to confuse young developers.

Reply:
I think that the main thing they should learn is how to **debug** the app. And it is much easier to debug a local python process than a process inside a running container, especially if you are a young padawan